### PR TITLE
💪  1.10.2

### PR DESCRIPTION
**Pull request by jspspike**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 💪 1.10.2
+
+- ### Fixes
+
+  - **Fixes bug preventing `wrangler publish` from deleting over 10000 keys - [ashleymichal], [issue/1398] [pull/1400]**
```
Comment (EverlastingBugstopper) [inline CHANGELOG.md:7]: need to add links here for the PRs, issues, and author - otherwise these won't go anywhere

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+    In 1.10.0 we removed the limit on bulk KV PUT and DELETE operations by batching the values but accidentally left situations where you would still run into a limit. Now with some refactoring all batched operations use the same logic and have the same behavior.
```
Comment (EverlastingBugstopper) [inline CHANGELOG.md:9]:
```suggestion
    In 1.10.0 we removed the limit on bulk KV PUT and DELETE operations by batching the values but missed batched deletes for Workers Sites. Now with some refactoring all batched operations use the same logic and have the same behavior.
```

Comment (EverlastingBugstopper) [reply CHANGELOG.md:9]: generally like to be as specific as possible with these 😄

File: CHANGELOG.md [hunk 3]
```diff
@@ -2033,11 +2041,11 @@ Wrangler 1.1.0 includes a number of improvements to documentation and project st
 
     [pull/285]: https://github.com/cloudflare/wrangler/pull/285
 
-  - **Refactor: Conditional per command in main - [ashleymical], [pull/279]**
+  - **Refactor: Conditional per command in main - [ashleymichal], [pull/279]**
 
     The `src/main.rs` file in Wrangler has been rewritten so that the layout of the file is easier to read.
 
-    [ashleymical]: https://github.com/ashleymical
+    [ashleymichal]: https://github.com/ashleymichal
```
Comment (EverlastingBugstopper) [inline CHANGELOG.md:2048]: good catches on these 😄

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 💪 1.10.2
+
+- ### Fixes
+
+  - **Fixes bug preventing `wrangler publish` from deleting over 10000 keys - [ashleymichal], [issue/1398] [pull/1400]**
+
+    In 1.10.0 we removed the limit on bulk KV PUT and DELETE operations by batching the values but missed batched deletes for Workers Sites. Now with some refactoring all batched operations use the same logic and have the same behavior.
```
Comment (signalnerve) [inline CHANGELOG.md:9]: little thing, but maybe we could link to 1.10.0/the specific change here? i have kind of a vague sense of what "bulk KV operations" mean but might be nice to improve the paper trail here

Comment (signalnerve) [reply CHANGELOG.md:9]: oh, saw this just got merged! feel free to ignore this

Comment (jspspike) [reply CHANGELOG.md:9]: Yeah sorry, I totally missed this but that's a good idea. Next release I'll add a suggestion to link the PR.

**Pull request merged**